packages/headless/jest.config.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
+  silent: true,
```

Comment:
Just removing some noise (slient == no console.log)
+ verbose = breakdown of every test suite one by one, easier to see when something fails IMO.